code/modules/mapping/map_template.dm

```diff
@@ -107,7 +107,7 @@
 	if(T.y+height > world.maxy)
 		return
 
-	var/list/border = block(locate(max(T.x-1, 1),			max(T.y-1, 1),			 T.z),
+	var/list/border = block(locate(max(T.x-1, 1),			max(T.y-1, 1),			 T.z),//kyler, this makes a block of turfs between the farthest possible turf corner and the closest?
 							locate(min(T.x+width+1, world.maxx),	min(T.y+height+1, world.maxy), T.z))
```

Comment:
This one make block of turfs between lower left-1 and upper right+1. outer border

Reply:
I wasn't really sure what key did but I don't pass program.type to tgui anymore so I changed it to id